deployments/cloudfoundry/buildpack/bin/supply

```diff
@@ -1,79 +1,86 @@
-#!/usr/bin/env python3
+#!/bin/bash
```

Comment:
would it make sense to use `/bin/sh` instead of `/bin/bash`? Perhaps not worth the effort.

Reply:
Possibly, but when what happens if/when the default shell isn't `bash`? Seems risky doesn't it?